Makefile

```diff
@@ -13,23 +13,18 @@ TOOLS_OUT := $(ROOT_OUT)/tools
 all: hypervisor devicemodel tools
 
 hypervisor:
-	cd $(T)/hypervisor; \
-	make HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE) clean; \
-	make HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE)
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE) clean; \
```

Comment:
good idea to use "-C".  can we remove "; \" at the end of the line?

Reply:
can't remove ';' because there is another line following '\' after that

Reply:
from below , seems "; \" are not needed.


+	make -C $(T)/tools/acrnlog OUT_DIR=$(TOOLS_OUT) install
+	make -C $(T)/tools/acrn-manager OUT_DIR=$(TOOLS_OUT) install
+	make -C $(T)/tools/acrntrace OUT_DIR=$(TOOLS_OUT) install

Reply:
Then need to remove both ";" and "\" together. 
Currently after this patch:
hypervisor:
	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE) clean; \
	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE)

Reply:
";\" are not needed with "make -C".
";\" are needed with "cd " because they have to be in the same statement.

please remove all unnecessary ";\" and ";"

Thanks,
Anthony

Reply:
";\\" are not needed with "make -C".
";\\" are needed with "cd " because they have to be in the same statement.

please remove all unnecessary ";\\" and ";"

Reply:
finally find out a way input '\\',